Firestore/Source/API/FIRFirestore.mm

```diff
@@ -329,6 +329,23 @@ + (void)enableLogging:(BOOL)logging {
   util::LogSetLevel(logging ? util::kLogLevelDebug : util::kLogLevelNotice);
 }
 
+- (void)useEmulatorWithHost:(NSString *)host port:(NSInteger)port {
+  if (!host.length) {
+    ThrowInvalidArgument("Host cannot be nil or empty.");
+  }
+  if (!_settings.isUsingDefaultHost) {
```

Comment:
In Android we decided to make this just a warning and let the `useEmulator` value win over what's in the settings.

Reply:
Done.